app/services/export_file_attachments.rb

```diff
@@ -0,0 +1,68 @@
+class ExportFileAttachments
+  attr_reader :source_file, :blob
+
+  TEMP_FILE_PATH = "foo.zip"
+  BULLETIN_LIST_URL = "https://www.apprenticeship.gov/about-us/legislation-regulations-guidance/bulletins/export?search=&category%5B0%5D=National%20Guideline%20Standards&category%5B1%5D=National%20Program%20Standards&category%5B2%5D=Occupations&page&_format=csv"
```

Comment:
Is this the source url for the StandardImport created here? I think I'm missing some context.

Reply:
This is the URL that we use to scrape the bulletins from.

Reply:
You shouldn't need it in this new service.